File: server/Timelogger/Entities/Project.cs
using System;
using Newtonsoft.Json;
using System.Collections.Generic;

namespace Timelogger.Entities
{
	public class Project : _EntityBase
	{
		[JsonProperty("id")]
        public Guid Id { get; set; }
		[JsonProperty("name")]
		public string Name { get; set; }
		[JsonProperty("description")]
		public string Description { get; set; }
		[JsonProperty("activity")]
		public ICollection<Activity> Activity { get; set; }
		[JsonProperty("status")]
		public string Status { get; set; }
		[JsonProperty("startDate")]
		public DateTime StartDate { get; set; }
		[JsonProperty("endDate")]
		public DateTime? EndDate { get; set; }
		[JsonProperty("currency")]
		public double Currency { get; set; }
			
	}
}

File: client/src/app/models/project.ts
//import { Activity } from "./activity";

export type ProjectStatus = 'completed' | 'pending' | 'failed';

// export interface Project{
//     id: string;
//     name: string;
//     description: string;
//     status: ProjectStatus;
//     startDate: number;
//     endDate: number;
//     currency: string;
// }

export interface Project {
    id: string;
    name: string;
    description: string;
    //activity?: Activity[] | null;
    status: ProjectStatus;
    startDate: Date;
    endDate: Date;
    currency: number;
}

//startDate: "2022-06-19T00:00:00+05:30"
